gendiff/utils.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Dict
```

Comment:
it's not utils module, rather the main project logic :) consider renaming to `diff` or `core`